src/ui/Controls/VideoPlayerContainer.cs

```diff
@@ -399,6 +400,30 @@ public void SetSubtitleText(string text, Paragraph p, Subtitle subtitle)
             }
         }
 
+        private SsaStyle _mpvStyle;
+        public void UpdateMpvStyle()
+        {
+            if (_mpvStyle != null)
```

Comment:
Is this null check + null assignment needed?

Reply:
I guess not, when I wrote it I thought the function would be used in more places and might require resetting the object.